packages/desktop/package.json

```diff
@@ -48,9 +48,9 @@
         "copy-webpack-plugin": "^11.0.0",
         "cross-env": "^7.0.2",
         "css-loader": "^6.7.3",
-        "electron": "22.3.27",
-        "electron-builder": "^24.0.0",
-        "electron-notarize": "^1.0.0",
+        "electron": "27.0.2",
+        "electron-builder": "^24.6.4",
```

Comment:
I'm not sure if this was in previous versions of `electron-builder`, but we may need to adjust our `electron-builder-config.js` so that we don't notarize the app twice (it's unclear what the default behavior of the [`notarize` option](https://www.electron.build/configuration/mac) is — it seems to wrap around `@electron/notarize`)

Reply:
We weren't signing twice, but now have moved the logic so electron-builder will sign for us

Reply:
@begonaalvarezd fyi